runtime/tests/resources/bind-panic/src/main.rs

```diff
@@ -1,13 +1,13 @@
 struct MyService;
 
-#[shuttle_service::async_trait]
-impl shuttle_service::Service for MyService {
-    async fn bind(mut self, _: std::net::SocketAddr) -> Result<(), shuttle_service::Error> {
+#[shuttle_runtime::async_trait]
+impl shuttle_runtime::Service for MyService {
+    async fn bind(mut self, _: std::net::SocketAddr) -> Result<(), shuttle_runtime::Error> {
         panic!("panic in bind");
     }
 }
 
-#[shuttle_service::main]
-async fn bind_panic() -> Result<MyService, shuttle_service::Error> {
+#[shuttle_runtime::main]
+async fn bind_panic() -> Result<MyService, shuttle_runtime::Error> {
     Ok(MyService)
 }
```

Comment:
This works here since we're not using any service integration from shuttle-service, but for those that do this won't be great. We'd have to add feature flags to runtime that adds a flag to runtime's dependency on shuttle-service. So it's probably best to pull out the integrations, like you suggested. Will we do that by making them individual crates, like we do for the resources?

Reply:
Yes, we will at least try to make them individual crates